tailwind.config.js

```diff
@@ -2,7 +2,15 @@ module.exports = {
   purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: theme => ({
+        'banner-img' :"url('../../public/bg-banner.jpg')"
+      }),
+      colors: {
+        'dk-blue': '#4F5D6C',
+        'hr-red-bar': '#BD6166'
```

Comment:
Sorry to go back and forth on this: can we match the Canada.ca colours for the background colour on the box?

Reply:
Changes made.